src/archanaNJun20/FibonacciNumbers.java

```diff
@@ -0,0 +1,27 @@
+/*Program 2: To print first n Fibonacci Numbers.
+Input n = 6
+Output = 1 1 2  3  5  8
+
+Input: n = 10 
+Output :  1 1 2  3  5  8 13  21 34 55*/
+package archanaNJun20;
+
+public class FibonacciNumbers {
+	void printFibonacciNumbers(int input){
+		int num1=1;
+		int num2=1;
+		int sum=0;
+		System.out.print("\n"+num1 + "  "+num2+" ");
+		for(int index=2;index<input;index++){
+			sum=num1+num2;
+			num1=num2;
+			num2=sum;
```

Comment:
you can calculate Fibonacci series using two variables only.
Hint: just like you have done in swap program.

Reply:
The code updated based on the comment. calculated the Fibonacci series without using the third variable.
thanks for your suggestion.please review and approve.

Reply:
Good job (Y)